libcst/tests/test_type_enforce.py

```diff
@@ -182,7 +182,7 @@ def test_basic_pass(self, value, expected_type) -> None:
             #   List[ -> means its invariant
             #     MyExampleClass -> has a subclass
             # does not allow List[List[MyExampleChildClass]]
-            ([[MyExampleChildClass()]], Iterable[List[MyExampleClass]]),
+            ([[MyExampleChildClass()]], Iterable[List[MyExampleClass]]),  # pyre-ignore
```

Comment:
Can you also add more detail about what's been ignored here? (Just like what you did for the other cases).

Reply:
Its the same as the above errors :\